File: nodejs/cmd_arg.js
const args = process.argv.slice(2);
const num1=parseFloat(args[0]);
const num2=parseFloat(args[1]);
if (isNaN(num1) || isNaN(num2)) {
    console.log('Invalid input. Please provide two numbers.');
} else {
    const sum=num1 + num2;
    console.log(`The sum of ${num1} and ${num2} is ${sum}`);
}

File: nodejs/cmd_arg.java
public class cmd_arg {
    public static void main(String[] args) {
        if (args.length < 2) {
            System.out.println("Please provide two numbers as command line arguments.");
            return;
        }

        try {
            int num1 = Integer.parseInt(args[0]);
            int num2 = Integer.parseInt(args[1]);
            int sum = num1 + num2;
            System.out.println("The sum is: " + sum);
        } catch (NumberFormatException e) {
            System.out.println("Invalid input. Please enter two valid numbers.");
        }
    }
}
